gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/EnforceFlowFinishDeadlineDagProc.java

```diff
@@ -58,7 +58,6 @@ protected void enforceDeadline(DagManagementStateStore dagManagementStateStore,
 
       dag.setFlowEvent(TimingEvent.FlowTimings.FLOW_RUN_DEADLINE_EXCEEDED);
       dag.setMessage("Flow killed due to exceeding SLA of " + flowFinishDeadline + " ms");
-      dagManagementStateStore.checkpointDag(dag);
```

Comment:
i was "checkpointing" them because i was changing "flowEvent" and "message" fields. but i realize these two fields are not persisted anyway.